template-files/android/AndroidManifest.xml

```diff
@@ -343,6 +343,12 @@
     </service>
 
     <!-- FCM -->
+    <service
+      android:name=".fcm.ExpoFcmMessagingService">
+      <intent-filter android:priority="1">
```

Comment:
Great job, sorry for not noticing it earlier! :+1: Do you think we should set the priority of the library's `FirebaseListenerService` to `-1` (so that we don't have to set the priority to `1` here)? Would it still work in `bare-expo` if we did this?

Reply:
I think set priority to -1 in `FirebaseListenerService` is a good idea :D And it should work in `bare-expo` ;)